assets/src/containers/DashboardAppBar.js

```diff
@@ -64,7 +63,7 @@ function DashboardAppBar (props) {
           </IconButton>
           <Button>
             <Typography variant='h6' color='inherit' className={classes.grow}>
-              <Link to={courseId} className={classes.homeButton}>My Learning Analytics: {courseName} </Link>
+              <Link to={{pathname:`/${courseId}`}} className={classes.homeButton}>My Learning Analytics: {courseName} </Link>
```

Comment:
This is will also lead the user to navigate to `/course/234555/` instead of `/course/234555/234555`